packages/desktop-client/src/components/reports/graphs/SpendingGraph.tsx

```diff
@@ -139,11 +155,15 @@ export function SpendingGraph({
     selection === 'average'
       ? data.intervalData[27].average
       : data.intervalData.reduce((a, b) =>
-          a.months[lastMonth][balanceTypeOp] <
-          b.months[lastMonth][balanceTypeOp]
+          a.months[selection === lastMonth ? lastMonth : lastYear][
```

Comment:
you could just use selection here
```suggestion
          a.months[selection][balanceTypeOp] <
```